pkg/oceancd/model/operator/install.go

```diff
@@ -0,0 +1,163 @@
+				PodLabels:        map[string]string{},
+				PodAnnotations:   map[string]string{},
+				Labels:           map[string]string{},
+				NodeSelector:     map[string]string{},
+				Tolerations:      []corev1.Toleration{},
+				ExtraEnv:         []corev1.EnvVar{},
+				ImagePullSecrets: []corev1.LocalObjectReference{},
+			},
+			Operator: configs.OceanCDOperatorConfig{
+				MetadataConfig: configs.MetadataConfig{
+					Labeled:   configs.Labeled{Labels: map[string]string{}},
+					Annotated: configs.Annotated{Annotations: map[string]string{}},
+				},
+				ExtraEnv:                  []corev1.EnvVar{},
+				NodeSelector:              map[string]string{},
+				Tolerations:               []corev1.Toleration{},
+				ImagePullSecrets:          []corev1.LocalObjectReference{},
+				ServiceAccountAnnotations: map[string]string{},
+			},
+		},
+		ArgoRolloutsConfig: configs.ArgoRolloutsConfig{
+			General: configs.ArgoRolloutsGeneralConfig{
+				Namespaced:         configs.Namespaced{Namespace: "argo-rollouts"},
+				Labeled:            configs.Labeled{Labels: map[string]string{}},
+				PodAnnotations:     map[string]string{},
+				PodLabels:          map[string]string{},
+				ServiceAnnotations: map[string]string{},
+				ServiceLabels:      map[string]string{},
+			},
+			Controller: configs.ArgoRolloutsControllerConfig{
+				Replicas:                 pointer.Int64(1),
+				NodeSelector:             map[string]string{},
+				Tolerations:              []corev1.Toleration{},
+				ExtraArgs:                map[string]string{},
+				ExtraEnv:                 []corev1.EnvVar{},
+				ImagePullSecrets:         []corev1.LocalObjectReference{},
+				ContainerSecurityContext: &corev1.SecurityContext{},
+				Probes: configs.Probes{
+					LivenessProbe: &corev1.Probe{
+						ProbeHandler: corev1.ProbeHandler{
+							HTTPGet: &corev1.HTTPGetAction{
+								Path: "/healthz",
+								Port: intstr.FromString("health"),
+							},
+						},
+						InitialDelaySeconds: 30,
+						PeriodSeconds:       20,
+						FailureThreshold:    3,
+						SuccessThreshold:    1,
+						TimeoutSeconds:      10,
+					},
+					ReadinessProbe: &corev1.Probe{
+						ProbeHandler: corev1.ProbeHandler{
+							HTTPGet: &corev1.HTTPGetAction{
+								Path: "/metrics",
+								Port: intstr.FromString("metrics"),
+							},
+						},
+						InitialDelaySeconds: 15,
+						PeriodSeconds:       5,
+						FailureThreshold:    3,
+						SuccessThreshold:    1,
+						TimeoutSeconds:      4,
+					},
+				},
+			},
+			Dashboard: configs.ArgoRolloutsDashboardConfig{
+				Enabled:                  false,
+				Replicas:                 pointer.Int64(1),
+				NodeSelector:             map[string]string{},
+				Tolerations:              []corev1.Toleration{},
+				ExtraEnv:                 []corev1.EnvVar{},
+				ImagePullSecrets:         []corev1.LocalObjectReference{},
+				ContainerSecurityContext: &corev1.SecurityContext{},
+			},
+		},
+	}
+}
+
+type InstallationPayload struct {
```

Comment:
why don't you like InstallationPayload?

Reply:
because I don't think it represent the data correctly
this is a payload for oceancd api to get OM manifests
I'm open to other suggestions than `OMManifestsRequest`